userland/examples/nrf51_aes/main.c

```diff
@@ -0,0 +1,204 @@
+#include <stdio.h>
```

Comment:
[file-level comment]

Is there actually anything nrf51-specific here? Could this instead be `examples/tests/aes/` ?